src/main/java/org/dbflute/intro/app/logic/playsql/migration/PlaysqlMigrationLogic.java

```diff
@@ -51,6 +52,11 @@
  */
 public class PlaysqlMigrationLogic {
 
+    // ===================================================================================
+    //                                                                          Definition
+    //                                                                          ==========
```

Comment:
suggest: comment e.g. "the pattern is same as DBFlute specification"

Reply:
Oh, I have the same opinion.